source/client/output_formatter_impl.cc

```diff
@@ -273,7 +273,9 @@ std::string FortioOutputFormatterImpl::formatProto(const nighthawk::client::Outp
   *fortio_output.mutable_requestedduration() = output.options().duration();
   auto actual_duration = getAverageExecutionDuration(output);
   fortio_output.set_actualduration(actual_duration.count());
-  fortio_output.set_jitter(output.options().has_jitter_uniform());
+  fortio_output.set_jitter(output.options().has_jitter_uniform() &&
```

Comment:
There's something I'm missing here. As far as I can tell, the jitter in options is basically just being used as a boolean.  If it's there (and now, if it's greater than 0), then we use jitter.  But where does the int value actually come into play?

Reply:
In Fortio jittering is a bool flag. On our side it's quite different, but we can emulate Fortio's version of it by using a certain configuration (10% of the qps from the top of my head).
So here we are more or less doing the best we can. Setting the right value here will make the Fortio UI display
wether jitter was set.

Reply:
Oh, and what might be confusing here, is that on our side, next to it being set, it must also have a value.